Clarify concepts documentation page (#8836)
traefik/traefik: docs/content/getting-started/concepts.md

![Decentralized Configuration](../assets/img/traefik-concepts-2.png)

It means that when a service is deployed, Traefik detects it immediately and updates the routing rules in real time.

> The inverse is also true: when you remove a service from your infrastructure, the route will disappear accordingly.

Reviewer: ```suggestion
Similarly, when you remove a service from your infrastructure, the route will disappear accordingly.
```
